### DISCUSSION TITLE
Feature request: Filter by approval count < N

### DISCUSSION THREAD

**Discussion by a-pride** [community/community#69721] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

It would be extremely helpful to have a filter to allow us to view pull requests with less than two approvals. 

I propose "Less than N approvals" as a new filter. Ex:
`less-than-n-approvals:2`

Currently filtering for our team is a manual process of applying "No more reviews needed" tag or "Ready to merge" tags which is easy to forget to do.

**Comment by a-pride** [comment]
Looks like -review:approved works please disregard :)

**Comment by MatthewRyanRead** [comment]
Other filters support ranges, so it could just be `approvals`.  I would like to look for things that I could merge (after optionally approving), so something like `approvals:1..2` would be ideal for repos that require 2 approvals. `-review:approved` includes those with 0 so far and excludes those with 2, so doesn't quite address my use case.
